ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/_cq_dialog/.content.xml

```diff
@@ -120,6 +120,14 @@
                                               jcr:primaryType="nt:unstructured"
                                               text="Form Data Model"
                                               value="formdatamodel"/>
+                                            <optionConnectormodel
+                                              jcr:primaryType="nt:unstructured"
+                                              text="Connector"
+                                              value="connector">
+                                              <granite:rendercondition
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="/libs/fd/af/dor/rendercondition/marketo"/>
```

Comment:
Why is the marketo render condition inside DOR ?

Reply:
changed.